db/migrate/20150427223923_create_notes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateNotes < ActiveRecord::Migration
+  def change
+    create_table :notes do |t|
+      t.string :body
+      t.references :company, index: true, foreign_key: true
```

Comment:
After adding indexes after the fact all morning long (during workshop), I really appreciate references now!
:sunglasses: